tests/commands/test_quotas_commands.py

```diff
@@ -0,0 +1,66 @@
+# tests/commands/test_quotas_commands.py
+
+from click.testing import CliRunner
+from mockito import when, verify
+from teaspoons_client import QuotaWithDetails, ApiException
+
+from terralab.commands import quotas_commands
+from tests.utils_for_tests import capture_logs
+
+
+def test_get_info_success(capture_logs, unstub):
+    test_pipeline_name = "test_pipeline"
+    test_quota_details = QuotaWithDetails(
+        pipeline_name=test_pipeline_name, quota_limit=1000, quota_consumed=300
+    )
+
+    when(quotas_commands.quotas_logic).get_user_quota(test_pipeline_name).thenReturn(
+        test_quota_details
+    )
+
+    runner = CliRunner()
+    result = runner.invoke(quotas_commands.quotas, ["get-info", test_pipeline_name])
+
+    assert result.exit_code == 0
+    verify(quotas_commands.quotas_logic).get_user_quota(test_pipeline_name)
+    assert test_pipeline_name in capture_logs.text
+    # quota limit
+    assert "1000" in capture_logs.text
```

Comment:
why not ask for the full string "Quota Limit: 1000"? it would make sure that the numbers didn't get mixed up in the output (e.g. we if accidentally report Quota Limit: 300, Quota Used: 1000, this test would still pass)